site/app/controllers/admin/PlagiarismController.php

```diff
@@ -746,44 +832,14 @@ public function saveNewPlagiarismConfiguration(string $new_or_edit, string $grad
             return new RedirectResponse($return_url);
         }
 
+        // We don't want to catch any errors here becasue if something goes wrong, we want to know about it as developers/sysadmins
```

Comment:
I'd remove this comment as we don't have this comment anywhere else we use `$em->flush()`.